pkg/grpc/portergrpc/installation.go

```diff
@@ -0,0 +1,113 @@
+package portergrpc
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	iGRPC "get.porter.sh/porter/gen/proto/go/porterapis/installation/v1alpha1"
+	"get.porter.sh/porter/pkg/porter"
+	"get.porter.sh/porter/pkg/tracing"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+func makeInstOptsLabels(labels map[string]string) []string {
```

Comment:
I assume `Inst` stands for instance? If so, I think it's worth a few extra keystrokes to ensure nobody else has to ask that question:

```suggestion
func makeInstanceOptsLabels(labels map[string]string) []string {
```